lib/both/router.coffee

```diff
@@ -1,3 +1,7 @@
+if Meteor.isClient
+	Session.set 'admin_bootstrap_loaded', false
```

Comment:
`.set` instead of `.setDefault` because then they won't be loaded when the page is reloaded and session is kept (i.e. Meteor restarted)